### ISSUE TITLE
Lights undim when video ends

### ISSUE THREAD

**Issue by scottcfuller** [mpolednik/script.kodi.hue.ambilight#60]
How to make the lights go back to 100% when the show automatically comes to an end.

**Comment by benstechtips** [comment]
I would like to see this feature too! :)

**Comment by hstamas** [comment]
It doesn't do that already? I have to admit that I rarely let a movie run to the very end without hitting stop at some point during the credits but I just assumed once the player reported that the video was over (ended/stopped) the plugin would turn the lights back on?

**Comment by scottcfuller** [comment]
Nope, doesn't do it.

**Comment by benstechtips** [comment]
Correct it does not. I agree that when you are watching a film, it is hardly ever left to run to the end, but when you're watching a TV show, for example, it is likely to happen.

**Comment by scottcfuller** [comment]
Correct

**Comment by dw20155** [comment]
I think this is s bug relating to the checking of the playlist length before a tv show or movie has ended

If you comment out these lines (138 and 139) of default.py which is:

        if self.playlistpos < self.playlistlen-1:
            return

then it works fine

perhaps it relates to playlistpos should that be playlistlen?

**Comment by tdb** [comment]
Yup, @dw20155 is right. `playlistpos` was added in from #18 in https://github.com/mpolednik/script.kodi.hue.ambilight/commit/c1d72cba41f667b4b24861ea31dc5e8d8d6c9a77#diff-e87f4dd869f897586689ccfd0fa643d5R92 but only partially removed by @mpolednik in https://github.com/mpolednik/script.kodi.hue.ambilight/commit/51dfb710a71e83493dc8a71b826a0a95446e91ee#diff-e87f4dd869f897586689ccfd0fa643d5L89. Given `playlistpos` no longer exists those two lines should just be removed.

I've created #70 to remove the broken lines.
